manifests/charts/gateway/values.yaml

```diff
@@ -139,3 +139,9 @@ volumes: []
 # A list of `VolumeMounts` added into the Gateway Pods. See
 # https://kubernetes.io/docs/concepts/storage/volumes/.
 volumeMounts: []
+
+# Configure this to a higher priority class in order to make sure your Istio gateway pods
+# will not be killed because of low priority class.
+# Refer to https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+# for more detail.
+priorityClassName: ""
```

Comment:
Thanks for the suggestion, but post-render doesn't work with ArgoCD (https://github.com/argoproj/argo-cd/issues/3698), and on the other hand I think it doesn't seem to hurt having this value in the chart because:
- It's a very commonly used field and other gateway charts already have it.
- It's straightforward